src/adapters/_node/response.ts

```diff
@@ -139,6 +138,8 @@ export const NodeResponse: {
       // Free up memory
       this.#body = undefined;
       this.#init = undefined;
+      this.#headersObj = undefined;
+      this.#responseObj = undefined;
```

Comment:
I'm unsure about these, but they made sense to be here, like in 

https://github.com/h3js/srvx/blob/b8a61e13ab03210c109126df93ae2153df032593/src/adapters/_node/response.ts#L166-L175